.github/workflows/bumpVersions.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Bump component versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      jira_id:
+        description: ticket reference for the requested update (OPSEXP-1234)
+        required: true
+  push:
+
+permissions:
+  contents: "write"
+  pull-requests: "write"
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        version:
+          - latest
+          - 7.3.N
+          - 7.2.N
+          - 7.1.N
+          - 7.0.N
+          #- community
+    env:
+      MATRIX_VALUES_OUTPUT: /tmp/${{ matrix.version }}-values.yaml
+      UPDATECLI_MANIFEST_FILE: ./updatecli.d/enterprise-manifest.tpl
+      JIRA_ID: ${{ inputs.jira_id }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Updatecli
+        uses: updatecli/updatecli-action@v2
+
+      - name: Prepare values file for ${{ matrix.version }}
+        working-directory: ./updatecli.d/
+        run: |
+          yq -e e 'explode(.) | .matrix["${{ matrix.version }}"]' supported-matrix.yaml \
+          | tee > "$MATRIX_VALUES_OUTPUT"
+
+      - name: Get today date for cache key
```

Comment:
because I don't have any other proper key, so falling back to the current date

anyway, I wanted to do some more tests before merging to see if it's really worth it

Reply:
cache removed as it was saving just nothing